install/updates/73-certmap.update

```diff
@@ -0,0 +1,27 @@
+# Configuration for Certificate Identity Mapping
+dn: cn=certmap,cn=ipa,cn=etc,$SUFFIX
+default:objectclass: top
+default:objectclass: nsContainer
+default:objectclass: ipaConfigObject
+default:objectclass: ipaCertMapContainer
+default:objectclass: ipaCertMapConfigObject
```

Comment:
You can't have 2 structural object classes (`nsContainer` and `ipaCertMapConfigObject`) in an entry. [RFC 4512, section 2.4.2](https://tools.ietf.org/html/rfc4512#section-2.4.2) says:
> An object or alias entry is characterized by precisely one structural object class superclass chain which has a single structural object class as the most subordinate object class. This structural object class is referred to as the structural object class of the entry.

I would define `ipaCertMapConfigObject` as auxiliary to fix this.